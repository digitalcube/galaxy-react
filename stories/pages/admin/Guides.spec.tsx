import { render } from '@testing-library/react';
import React from 'react';
import { Default as Guides } from './Guides.stories';

describe('Guides', () => {
  it('to match snapshot(default)', () => {
    const { asFragment } = render(<Guides {...(Guides.args as any)} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
