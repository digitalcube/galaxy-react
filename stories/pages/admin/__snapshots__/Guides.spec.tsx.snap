// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Guides to match snapshot(default) 1`] = `
<DocumentFragment>
  <div
    class="d-flex flex-column min-vh-100 shifter-dashboard"
  >
    <nav
      class="navbar navbar-expand-lg"
    >
      <a
        class="navbar-brand"
        href="/"
      >
        <img
          alt="Shifter"
          height="32"
          width="24"
        />
      </a>
      <form
        class="form-inline my-2 my-lg-0 flex-grow-1 d-flex position-relative form-search"
      >
        <div
          class="input-group"
        >
          <svg
            class="fas fa-search form-search-icon position-absolute"
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 512 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M505 442.7L405.3 343c-4.5-4.5-10.6-7-17-7H372c27.6-35.3 44-79.7 44-128C416 93.1 322.9 0 208 0S0 93.1 0 208s93.1 208 208 208c48.3 0 92.7-16.4 128-44v16.3c0 6.4 2.5 12.5 7 17l99.7 99.7c9.4 9.4 24.6 9.4 33.9 0l28.3-28.3c9.4-9.4 9.4-24.6.1-34zM208 336c-70.7 0-128-57.2-128-128 0-70.7 57.2-128 128-128 70.7 0 128 57.2 128 128 0 70.7-57.2 128-128 128z"
            />
          </svg>
          <input
            aria-label="Search"
            class="form-control"
            placeholder="search"
            type="text"
            value=""
          />
        </div>
      </form>
      <button
        aria-label="Toggle navigation"
        class="navbar-toggler"
        type="button"
      >
        <span
          class="navbar-toggler-icon"
        />
      </button>
      <div
        class="navbar-collapse flex-grow-0 collapse"
      >
        <ul
          class="ml-auto navbar-nav nav"
        >
          <li
            class="active nav-item"
          >
            <a
              class="rounded font-weight-bold nav-link"
              to="#"
            >
              Sites
            </a>
          </li>
          <li
            class="nav-item"
          >
            <a
              class="rounded font-weight-bold nav-link"
              to="#"
            >
              Teams
            </a>
          </li>
          <li
            class="nav-item"
          >
            <a
              class="rounded font-weight-bold nav-link"
              to="#"
            >
              Guides
            </a>
          </li>
          <li
            class="dropdown nav-item"
          >
            <a
              aria-expanded="false"
              aria-haspopup="true"
              class="nav-link font-weight-bold dropdown-toggle nav-link"
              data-toggle="dropdown"
              href="#"
              role="button"
            >
              <img
                alt="Gravatar for test@example.com"
                class="react-gravatar p-0  rounded-circle"
                height="32"
                src="//www.gravatar.com/avatar/55502f40dc8b7c769880b10874abc9d0?d=retro&r=g&s=32"
                srcset="//www.gravatar.com/avatar/55502f40dc8b7c769880b10874abc9d0?d=retro&r=g&s=64 2x"
                width="32"
              />
            </a>
            <div
              aria-hidden="true"
              class="dropdown-menu dropdown-menu-right"
              role="menu"
              tabindex="-1"
            >
              <button
                class="dropdown-item"
                role="menuitem"
                tabindex="0"
                type="button"
              >
                <a
                  class="dropdown-item px-3 py-2 d-flex align-items-center justify-content-between"
                  href="#"
                  to="#"
                >
                  Account
                  <svg
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 448 512"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M224 256c70.7 0 128-57.3 128-128S294.7 0 224 0 96 57.3 96 128s57.3 128 128 128zm89.6 32h-16.7c-22.2 10.2-46.9 16-72.9 16s-50.6-5.8-72.9-16h-16.7C60.2 288 0 348.2 0 422.4V464c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48v-41.6c0-74.2-60.2-134.4-134.4-134.4z"
                    />
                  </svg>
                </a>
              </button>
              <button
                class="dropdown-item"
                role="menuitem"
                tabindex="0"
                type="button"
              >
                <a
                  class="dropdown-item px-3 py-2 d-flex align-items-center justify-content-between"
                  href="#"
                  to="#"
                >
                  Log out 
                  <svg
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 512 512"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M497 273L329 441c-15 15-41 4.5-41-17v-96H152c-13.3 0-24-10.7-24-24v-96c0-13.3 10.7-24 24-24h136V88c0-21.4 25.9-32 41-17l168 168c9.3 9.4 9.3 24.6 0 34zM192 436v-40c0-6.6-5.4-12-12-12H96c-17.7 0-32-14.3-32-32V160c0-17.7 14.3-32 32-32h84c6.6 0 12-5.4 12-12V76c0-6.6-5.4-12-12-12H96c-53 0-96 43-96 96v192c0 53 43 96 96 96h84c6.6 0 12-5.4 12-12z"
                    />
                  </svg>
                </a>
              </button>
            </div>
          </li>
        </ul>
      </div>
    </nav>
    <div
      class="mx-auto w-100 shifter-guides"
    >
      <h1
        class="guide-heading-1 font-weight-bold"
      >
        Guides
      </h1>
      <div
        class="mt-4 container-fluid"
      >
        <div
          class="row"
        >
          <div
            class="col px-0"
          >
            <a
              class="d-block"
              href="#"
            >
              <img
                alt="Introducing PHP Error Logs"
                class="img-fluid"
                height="180"
                src="/images/guide-banner.png"
                width="1134"
              />
            </a>
          </div>
        </div>
      </div>
      <div
        class="shifter-guides"
      >
        <div
          class="container-fluid mb-4 pb-4 px-0 guides-list guides-list-lg"
        >
          <div
            class="row"
          >
            <div
              class="col-md-4"
            >
              <div
                class="pt-4 px-4 h-100 position-relative guide-list-item"
              >
                <div
                  class="guides-list-icon"
                >
                  <i
                    aria-hidden="true"
                  >
                    <svg
                      class="d-block"
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      stroke-width="0"
                      viewBox="0 0 384 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M224 136V0H24C10.7 0 0 10.7 0 24v464c0 13.3 10.7 24 24 24h336c13.3 0 24-10.7 24-24V160H248c-13.2 0-24-10.8-24-24zm160-14.1v6.1H256V0h6.1c6.4 0 12.5 2.5 17 7l97.9 98c4.5 4.5 7 10.6 7 16.9z"
                      />
                    </svg>
                  </i>
                </div>
                <h3
                  class="mt-4 font-weight-bold guide-list-title"
                >
                  Read the docs
                </h3>
                <div
                  class="position-absolute guide-list-anchor"
                >
                  <a
                    class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-secondary"
                    href="https://getshifter.io"
                    rel="noopener"
                    target="_blank"
                  >
                    Explore Shifter docs
                    <i
                      aria-hidden="true"
                      class="pl-2 larger-icon"
                    >
                      <svg
                        class="d-block"
                        fill="currentColor"
                        height="1em"
                        stroke="currentColor"
                        stroke-width="0"
                        viewBox="0 0 320 512"
                        width="1em"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <path
                          d="M296 160H180.6l42.6-129.8C227.2 15 215.7 0 200 0H56C44 0 33.8 8.9 32.2 20.8l-32 240C-1.7 275.2 9.5 288 24 288h118.7L96.6 482.5c-3.6 15.2 8 29.5 23.3 29.5 8.4 0 16.4-4.4 20.8-12l176-304c9.3-15.9-2.2-36-20.7-36z"
                        />
                      </svg>
                    </i>
                  </a>
                </div>
              </div>
            </div>
            <div
              class="col-md-4"
            >
              <div
                class="pt-4 px-4 h-100 position-relative guide-list-item"
              >
                <div
                  class="guides-list-icon"
                >
                  <i
                    aria-hidden="true"
                  >
                    <svg
                      class="d-block"
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      stroke-width="0"
                      viewBox="0 0 448 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M424.4 214.7L72.4 6.6C43.8-10.3 0 6.1 0 47.9V464c0 37.5 40.7 60.1 72.4 41.3l352-208c31.4-18.5 31.5-64.1 0-82.6z"
                      />
                    </svg>
                  </i>
                </div>
                <h3
                  class="mt-4 font-weight-bold guide-list-title"
                >
                  Watch the tutorials
                </h3>
                <p
                  class="mb-0 guide-list-excerpt"
                >
                  Get the most from your Jamstack WordPress site
                </p>
                <div
                  class="position-absolute guide-list-anchor"
                >
                  <a
                    class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-secondary"
                    href="https://getshifter.io"
                    rel="noopener"
                    target="_blank"
                  >
                    Start Watching
                    <i
                      aria-hidden="true"
                      class="pl-2 larger-icon"
                    >
                      <svg
                        class="d-block"
                        fill="currentColor"
                        height="1em"
                        stroke="currentColor"
                        stroke-width="0"
                        viewBox="0 0 320 512"
                        width="1em"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <path
                          d="M296 160H180.6l42.6-129.8C227.2 15 215.7 0 200 0H56C44 0 33.8 8.9 32.2 20.8l-32 240C-1.7 275.2 9.5 288 24 288h118.7L96.6 482.5c-3.6 15.2 8 29.5 23.3 29.5 8.4 0 16.4-4.4 20.8-12l176-304c9.3-15.9-2.2-36-20.7-36z"
                        />
                      </svg>
                    </i>
                  </a>
                </div>
              </div>
            </div>
            <div
              class="col-md-4"
            >
              <div
                class="pt-4 px-4 h-100 position-relative guide-list-item"
              >
                <div
                  class="guides-list-icon"
                >
                  <i
                    aria-hidden="true"
                  >
                    <svg
                      class="d-block"
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      stroke-width="0"
                      viewBox="0 0 512 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M507.73 109.1c-2.24-9.03-13.54-12.09-20.12-5.51l-74.36 74.36-67.88-11.31-11.31-67.88 74.36-74.36c6.62-6.62 3.43-17.9-5.66-20.16-47.38-11.74-99.55.91-136.58 37.93-39.64 39.64-50.55 97.1-34.05 147.2L18.74 402.76c-24.99 24.99-24.99 65.51 0 90.5 24.99 24.99 65.51 24.99 90.5 0l213.21-213.21c50.12 16.71 107.47 5.68 147.37-34.22 37.07-37.07 49.7-89.32 37.91-136.73zM64 472c-13.25 0-24-10.75-24-24 0-13.26 10.75-24 24-24s24 10.74 24 24c0 13.25-10.75 24-24 24z"
                      />
                    </svg>
                  </i>
                </div>
                <h3
                  class="mt-4 font-weight-bold guide-list-title"
                >
                  Explore extensions
                </h3>
                <p
                  class="mb-0 guide-list-excerpt"
                >
                  Get the most from your Jamstack WordPress site
                </p>
                <div
                  class="position-absolute guide-list-anchor"
                >
                  <a
                    class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-secondary"
                    href="https://getshifter.io"
                    rel="noopener"
                    target="_blank"
                  >
                    Explore Jamstack extensions
                    <i
                      aria-hidden="true"
                      class="pl-2 larger-icon"
                    >
                      <svg
                        class="d-block"
                        fill="currentColor"
                        height="1em"
                        stroke="currentColor"
                        stroke-width="0"
                        viewBox="0 0 320 512"
                        width="1em"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <path
                          d="M296 160H180.6l42.6-129.8C227.2 15 215.7 0 200 0H56C44 0 33.8 8.9 32.2 20.8l-32 240C-1.7 275.2 9.5 288 24 288h118.7L96.6 482.5c-3.6 15.2 8 29.5 23.3 29.5 8.4 0 16.4-4.4 20.8-12l176-304c9.3-15.9-2.2-36-20.7-36z"
                        />
                      </svg>
                    </i>
                  </a>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <h2
        class="guide-heading-2 font-weight-bold"
      >
        Popular articles
      </h2>
      <div
        class="container-fluid px-0 guides-list guides-list-md"
      >
        <div
          class="row"
        >
          <div
            class="col-md-4"
          >
            <div
              class="pt-4 px-4 h-100 position-relative guide-list-item"
            >
              <div
                class="mb-2 guide-list-genre"
              >
                Developers / Domain
              </div>
              <h3
                class="mb-2 guide-list-title font-weight-bold"
              >
                Custom domains on Shifter
              </h3>
              <p
                class="mb-0 guide-list-excerpt"
              >
                This article contains step by step guide for adding and assigning your custom domain to your Shifter site.
              </p>
              <div
                class="position-absolute guide-list-anchor"
              >
                <a
                  class="px-3 py-0 rounded btn btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold"
                  href="#"
                >
                  Read this article
                  <i
                    aria-hidden="true"
                    class="pl-2 larger-icon"
                  >
                    <svg
                      class="d-block"
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      stroke-width="0"
                      viewBox="0 0 320 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M296 160H180.6l42.6-129.8C227.2 15 215.7 0 200 0H56C44 0 33.8 8.9 32.2 20.8l-32 240C-1.7 275.2 9.5 288 24 288h118.7L96.6 482.5c-3.6 15.2 8 29.5 23.3 29.5 8.4 0 16.4-4.4 20.8-12l176-304c9.3-15.9-2.2-36-20.7-36z"
                      />
                    </svg>
                  </i>
                </a>
              </div>
            </div>
          </div>
          <div
            class="col-md-4"
          >
            <div
              class="pt-4 px-4 h-100 position-relative guide-list-item"
            >
              <div
                class="mb-2 guide-list-genre"
              >
                Developers / Domain
              </div>
              <h3
                class="mb-2 guide-list-title font-weight-bold"
              >
                Custom domains on Shifter Custom domains on Shifter
              </h3>
              <p
                class="mb-0 guide-list-excerpt"
              >
                This article contains step by step guide for adding and assigning your custom domain to your Shifter site.
              </p>
              <div
                class="position-absolute guide-list-anchor"
              >
                <a
                  class="px-3 py-0 rounded btn btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold"
                  href="#"
                >
                  Read this article
                  <i
                    aria-hidden="true"
                    class="pl-2 larger-icon"
                  >
                    <svg
                      class="d-block"
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      stroke-width="0"
                      viewBox="0 0 320 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M296 160H180.6l42.6-129.8C227.2 15 215.7 0 200 0H56C44 0 33.8 8.9 32.2 20.8l-32 240C-1.7 275.2 9.5 288 24 288h118.7L96.6 482.5c-3.6 15.2 8 29.5 23.3 29.5 8.4 0 16.4-4.4 20.8-12l176-304c9.3-15.9-2.2-36-20.7-36z"
                      />
                    </svg>
                  </i>
                </a>
              </div>
            </div>
          </div>
          <div
            class="col-md-4"
          >
            <div
              class="pt-4 px-4 h-100 position-relative guide-list-item"
            >
              <div
                class="mb-2 guide-list-genre"
              >
                Developers / Domain
              </div>
              <h3
                class="mb-2 guide-list-title font-weight-bold"
              >
                Custom domains on Shifter
              </h3>
              <p
                class="mb-0 guide-list-excerpt"
              >
                This article contains step by step guide for adding and assigning your custom domain to your Shifter site.
              </p>
              <div
                class="position-absolute guide-list-anchor"
              >
                <a
                  class="px-3 py-0 rounded btn btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold"
                  href="#"
                >
                  Read this article
                  <i
                    aria-hidden="true"
                    class="pl-2 larger-icon"
                  >
                    <svg
                      class="d-block"
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      stroke-width="0"
                      viewBox="0 0 320 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M296 160H180.6l42.6-129.8C227.2 15 215.7 0 200 0H56C44 0 33.8 8.9 32.2 20.8l-32 240C-1.7 275.2 9.5 288 24 288h118.7L96.6 482.5c-3.6 15.2 8 29.5 23.3 29.5 8.4 0 16.4-4.4 20.8-12l176-304c9.3-15.9-2.2-36-20.7-36z"
                      />
                    </svg>
                  </i>
                </a>
              </div>
            </div>
          </div>
          <div
            class="col-md-4"
          >
            <div
              class="pt-4 px-4 h-100 position-relative guide-list-item"
            >
              <div
                class="mb-2 guide-list-genre"
              >
                Developers / Domain
              </div>
              <h3
                class="mb-2 guide-list-title font-weight-bold"
              >
                Custom domains on Shifter
              </h3>
              <p
                class="mb-0 guide-list-excerpt"
              >
                This article contains step by step guide for adding and assigning your custom domain to your Shifter site.
              </p>
              <div
                class="position-absolute guide-list-anchor"
              >
                <a
                  class="px-3 py-0 rounded btn btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold"
                  href="#"
                >
                  Read this article
                  <i
                    aria-hidden="true"
                    class="pl-2 larger-icon"
                  >
                    <svg
                      class="d-block"
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      stroke-width="0"
                      viewBox="0 0 320 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M296 160H180.6l42.6-129.8C227.2 15 215.7 0 200 0H56C44 0 33.8 8.9 32.2 20.8l-32 240C-1.7 275.2 9.5 288 24 288h118.7L96.6 482.5c-3.6 15.2 8 29.5 23.3 29.5 8.4 0 16.4-4.4 20.8-12l176-304c9.3-15.9-2.2-36-20.7-36z"
                      />
                    </svg>
                  </i>
                </a>
              </div>
            </div>
          </div>
          <div
            class="col-md-4"
          >
            <div
              class="pt-4 px-4 h-100 position-relative guide-list-item"
            >
              <div
                class="mb-2 guide-list-genre"
              >
                Developers / Domain
              </div>
              <h3
                class="mb-2 guide-list-title font-weight-bold"
              >
                Custom domains on Shifter
              </h3>
              <p
                class="mb-0 guide-list-excerpt"
              >
                This article contains step by step guide for adding and assigning your custom domain to your Shifter site. This article contains step by step guide for adding and assigning your custom domain to your Shifter site.
              </p>
              <div
                class="position-absolute guide-list-anchor"
              >
                <a
                  class="px-3 py-0 rounded btn btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold"
                  href="#"
                >
                  Read this article
                  <i
                    aria-hidden="true"
                    class="pl-2 larger-icon"
                  >
                    <svg
                      class="d-block"
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      stroke-width="0"
                      viewBox="0 0 320 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M296 160H180.6l42.6-129.8C227.2 15 215.7 0 200 0H56C44 0 33.8 8.9 32.2 20.8l-32 240C-1.7 275.2 9.5 288 24 288h118.7L96.6 482.5c-3.6 15.2 8 29.5 23.3 29.5 8.4 0 16.4-4.4 20.8-12l176-304c9.3-15.9-2.2-36-20.7-36z"
                      />
                    </svg>
                  </i>
                </a>
              </div>
            </div>
          </div>
          <div
            class="col-md-4"
          >
            <div
              class="px-4 py-4 d-flex flex-column justify-content-center h-100 question"
            >
              <h3
                class="mb-0 font-weight-bold question-title"
              >
                Questions?
              </h3>
              <div
                class="mt-2 question-description"
              >
                Our team is happy to help
              </div>
              <div
                class="mt-4 question-anchor"
              >
                <a
                  class="px-3 py-0 rounded btn btn-bordered btn-smaller border-0 d-inline-flex align-items-center font-weight-bold"
                  href="#"
                >
                  Contact Support
                  <i
                    aria-hidden="true"
                    class="pl-2 larger-icon"
                  >
                    <svg
                      class="d-block"
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      stroke-width="0"
                      viewBox="0 0 512 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M256 32C114.6 32 0 125.1 0 240c0 49.6 21.4 95 57 130.7C44.5 421.1 2.7 466 2.2 466.5c-2.2 2.3-2.8 5.7-1.5 8.7S4.8 480 8 480c66.3 0 116-31.8 140.6-51.4 32.7 12.3 69 19.4 107.4 19.4 141.4 0 256-93.1 256-208S397.4 32 256 32z"
                      />
                    </svg>
                  </i>
                </a>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <footer
      class="text-center"
    >
      <p
        class="m-0 common-footer-copyright"
      >
        <small
          class="d-flex align-items-center justify-content-center"
        >
          ©2021 Shifter | A Product by 
          <a
            href="https://www.digitalcube.jp/"
            rel="nofollow noopener"
            title="DigitalCube"
          >
            <img
              alt=""
              class="d-block"
              height="15"
              width="77"
            />
          </a>
        </small>
      </p>
    </footer>
  </div>
</DocumentFragment>
`;
